packages/vscode-extension/src/devices/AndroidEmulatorDevice.ts

```diff
@@ -475,7 +496,12 @@ export async function listEmulators() {
   );
 }
 
-export function removeEmulator(avdId: string) {
+export async function removeEmulator(avdId: string) {
+  // ensure to kill emulator process before removing avd files used by that process
+  if (Platform.OS == "windows") {
+    await AndroidEmulatorDevice.ensureOldEmulatorProcessExited(avdId);
```

Comment:
Can we use `this` instead of `AndroidEmulatorDevice`? Statics should be available there IIRC.

Reply:
No we can't. Function `removeEmulator` is not a member of `AndroidEmulatorDevice` and in `bootDevice` case  I found: "Static members are not directly accessible using the [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this) keyword from non-static methods."